tests/testthat/test-cols_nanoplot.R

```diff
@@ -33,3 +33,35 @@ test_that("cols_nanoplot() works without error ", {
       fmt_number()
   })
 })
+
+test_that("check cols_nanoplot is applied gt_group", {
```

Comment:
@thebioengineer this one doesn't work with the same way of testing as the others because you get an error with tidyselect but they look the same